ssha/cli.py

```diff
@@ -43,3 +44,8 @@ def main():
 
     except KeyboardInterrupt:
         pass
+    except Exception as e:
+        if args.verbose:
+            print(traceback.format_exc())
```

Comment:
Thoughts on printing the exception verses doing `raise`?

If you prefer printing it, then should it also `return 1` here so the program exits with an error code?

Reply:
I think it's not user-friendly to crash with exception. Agreed, I should return a non-zero code

Reply:
Isn't printing the exception and then exiting with non-zero the same as crashing, as far as a user can see? (I can't remember what `traceback.format_exc()` looks like but I'm thinking it looks the same as what happens when Python crashes with an exception).